pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -466,20 +470,31 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1alpha1.PipelineRun) er
 		if rprt == nil {
 			continue
 		}
+		if rprt.IsRunOnConflicted {
+			rprt.TaskRun, err = c.createRunOnConflictedTaskRun(rprt, pr)
```

Comment:
Do we really need to create the TaskRun here? 
This TaskRun does not meet the precondition for being scheduled because it depends on a condition (as defined in runOn) that did not happen. And TaskRuns that depended on this one will never be scheduled because their predecessor in the DAG did not run.

Reply:
Answered above ⬆️ , once again, we can drop creating taskrun here but would be helpful to have one.